### PR TITLE
added an option for debug

### PR DESCRIPTION
**Pull request by harelon**

### CODE REVIEW

**Comment by bazulay** [comment]
This is a wrong approach,
1 - the configuration should not be called debug , as this is not debugging its simply storing the images
2 - the directory in which you store the images should come from configuration
3 - since the risk is that the disk is filled up (and the raspberriPy may stop functioning), you should check for available space before storing every image ... it will slow down
:-1:

**Comment by harelon** [comment]
This is how Udi told me to debug the program so the java can debug their path calculation to know by which frame they drove

**Comment by bazulay** [comment]
Read again the comments above, your response has nothing to do with the comments.
The fact that you want to use it for debug does not mean it needs to be called debug, a configuration name should reflect the actual functionality, and in this case, the functionality is storing the images.
+ there are 2 other comments you need to address in my previous review

**Comment by harelon** [comment]
I will fix and commit again, i will use a cyclic buffer for the place to store the images

File: service/rcv_server.conf [hunk 1]
```diff
@@ -12,7 +12,9 @@ frame_handler=frc_2019
 frame_handlers_dir=frame_handlers
 frame_threshold=230
 show=True
-debug=True
+save_camera_roll=True
+buffer_size=1000
+save_camera_roll_dir=/opt/rcv/camera_logs
```
Comment (bazulay) [inline service/rcv_server.conf]: Its not camera_logs ... it is more like  frames or images

**Pull request closed without merging**